.travis.yml

```diff
@@ -6,6 +6,8 @@ install:
   - pip install flake8
```

Comment:
Perhaps move this to `requirements-dev.txt` as well and remove this line from here.